doc/integrator/docker.rst

```diff
@@ -72,6 +72,17 @@ This will remove::
 * Images with version on name as `<none>`.
 * Unnamed dangling volumes.
 
+To remove unused volumes created by possibly unconsistent volume declarations
+in the ``Dockerfile`` compared to ``docker-compose.yml``:
+
+.. prompt:: bash
+
+    #!/bin/bash
+    for vol in $(docker volume ls | awk '{print $2}' | grep -v VOLUME)
+    do
+      docker volume rm $vol
+    done
```

Comment:
Why not using?
```
docker system prune --volumes 
```

Reply:
I have added the tip, as mentioned in https://techan.fr/docker-chasse-a-lespace-perdu-gare-aux-volumes.html and used by @yjacolin for customers that recently had disk space issues. I have no clue if both commands do the same thing.

@yjacolin what do you think? Have customers used the prune command as well?

Reply:
@yjacolin Oops, I should have read your next comment before posting :P